jenkins/python3-tests.sh

```diff
@@ -29,13 +24,10 @@ pip install -I -r test-requirements.txt
 python3 setup.py build
 python3 setup.py build_ext --inplace
 
-# not using "setup.py nosetests" yet
-# since they need a running candlepin
+export PYTHONPATH=src/
 
-# Run just the unit tests, functional needs a running candlepin
-#pushd test/unit
 # make sure we have a dbus session for the dbus tests
-dbus-run-session nosetests --with-xunit --with-cover --cover-package rhsm --cover-package subscription_manager --cover-erase
+dbus-run-session coverage run -m pytest test/
```

Comment:
Well, we are planning on enabling them in a near future. Because running `dbus-run-session` means little to no difference of time/resources, I thought we may just leave it here. On the other hand, we can remove it now and put it back in once the dbus tests are corrected; we just have to remember to do that.